### PR TITLE
Update the plugin BOM to the latest version of 2.222.x

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Thanks to @jglick and @timja , there is now a BOM for the 2.190.x baseline: https://github.com/jenkinsci/bom/releases/tag/bom-7 . It is still behind the JFR's target, but it is still  good update. 

P.S: https://github.com/jenkinsci/bom/pull/214 for the 2.222.x one

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
Oops, it is barely related to this PR though

```
[2020-04-13T08:47:07.648Z] [INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (display-info) @ payload ---

[2020-04-13T08:47:07.648Z] [INFO] Adding ignore: module-info

[2020-04-13T08:47:08.462Z] [WARNING] Rule 2: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:

[2020-04-13T08:47:08.462Z] Failed while enforcing RequireUpperBoundDeps. The error(s) are [

[2020-04-13T08:47:08.462Z] Require upper bound dependencies error for org.jenkins-ci:version-number:1.6 paths to dependency are:

[2020-04-13T08:47:08.462Z] +-io.jenkins.jenkinsfile-runner:payload:1.0-beta-12-SNAPSHOT

[2020-04-13T08:47:08.462Z]   +-io.jenkins.jenkinsfile-runner:payload-dependencies:1.0-beta-12-SNAPSHOT

[2020-04-13T08:47:08.462Z]     +-org.jenkins-ci.main:jenkins-core:2.222.1

[2020-04-13T08:47:08.462Z]       +-org.jenkins-ci:version-number:1.6

[2020-04-13T08:47:08.462Z] and

[2020-04-13T08:47:08.462Z] +-io.jenkins.jenkinsfile-runner:payload:1.0-beta-12-SNAPSHOT

[2020-04-13T08:47:08.462Z]   +-io.jenkins.jenkinsfile-runner:setup:1.0-beta-12-SNAPSHOT

[2020-04-13T08:47:08.462Z]     +-io.jenkins.jenkinsfile-runner:bootstrap:1.0-beta-12-SNAPSHOT

[2020-04-13T08:47:08.462Z]       +-org.jenkins-ci:version-number:1.7
```

File: bootstrap/pom.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
     <dependency>
       <groupId>org.jenkins-ci</groupId>
       <artifactId>version-number</artifactId>
-      <version>1.7</version>
+      <version>1.6</version>
```
Comment (timja) [inline bootstrap/pom.xml]: this should come from the jenkins bom

Comment (oleg-nenashev) [reply bootstrap/pom.xml]: Yes. This is what I am checking right now

Comment (oleg-nenashev) [reply bootstrap/pom.xml]: @timja BOM does not include it at the moment. https://github.com/jenkinsci/jenkins/blob/master/bom/pom.xml . Maybe a quick fix

**Review by jglick — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
Integration tests now explode with a weird error. Most likely it happens due to the collision between the BOM and versions defined in the integration test. I should switch all (or almost all) integration tests to use the vanilla image instead of CWP....


```
[2020-04-13T14:42:04.281Z] 	at java.lang.Thread.run(Thread.java:748)
[2020-04-13T14:42:05.100Z] 2020-04-13 14:42:04.848+0000 [id=25]	SEVERE	jenkins.InitReactorRunner$1#onTaskFailed: Failed Inspecting plugin /usr/share/jenkins/ref/plugins/script-security.hpi
[2020-04-13T14:42:05.100Z] src '/usr/share/jenkins/ref/plugins/script-security.hpi' doesn't exist.
[2020-04-13T14:42:05.100Z] 	at org.apache.tools.ant.taskdefs.Expand.execute(Expand.java:148)
[2020-04-13T14:42:05.100Z] 	at hudson.ClassicPluginStrategy.unzipExceptClasses(ClassicPluginStrategy.java:556)
[2020-04-13T14:42:05.100Z] 	at hudson.ClassicPluginStrategy.explode(ClassicPluginStrategy.java:481)
[2020-04-13T14:42:05.100Z] Caused: java.io.IOException: Failed to expand /usr/share/jenkins/ref/plugins/script-security.hpi
[2020-04-13T14:42:05.100Z] 	at hudson.ClassicPluginStrategy.explode(ClassicPluginStrategy.java:484)
[2020-04-13T14:42:05.100Z] 	at hudson.ClassicPluginStrategy.createPluginWrapper(ClassicPluginStrategy.java:175)
[2020-04-13T14:42:05.100Z] 	at hudson.PluginManager$1$3$1.run(PluginManager.java:433)
[2020-04-13T14:42:05.100Z] 	at org.jvnet.hudson.reactor.TaskGraphBuilder$TaskImpl.run(TaskGraphBuilder.java:169)
[2020-04-13T14:42:05.100Z] 	at org.jvnet.hudson.reactor.Reactor.runTask(Reactor.java:296)
[2020-04-13T14:42:05.100Z] 	at jenkins.model.Jenkins$5.runTask(Jenkins.java:1133)
[2020-04-13T14:42:05.100Z] 	at org.jvnet.hudson.reactor.Reactor$2.run(Reactor.java:214)
[2020-04-13T14:42:05.100Z] 	at org.jvnet.hudson.reactor.Reactor$Node.run(Reactor.java:117)
[2020-04-13T14:42:05.100Z] 	at jenkins.security.ImpersonatingExecutorService$1.run(ImpersonatingExecutorService.java:59)
[2020-04-13T14:42:05.100Z] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[2020-04-13T14:42:05.100Z] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[2020-04-13T14:42:05.100Z] 	at java.lang.Thread.run(Thread.java:748)
[2020-04-13T14:42:11.557Z] 2020-04-13 14:42:10.438+0000 [id=24]	WARNING	h.ExtensionFinder$GuiceFinder$FaultTolerantScope$1#error: Failed to instantiate Key[type=org.jenkinsci.plugins.workflow.cps.SandboxContinuable$ScriptApprovalNote$DescriptorImpl, annotation=[none]]; skipping this component
[2020-04-13T14:42:11.557Z] java.lang.IllegalAccessError: tried to access class org.jenkinsci.plugins.workflow.cps.SandboxContinuable$ScriptApprovalNote from class org.jenkinsci.plugins.workflow.cps.SandboxContinuable$ScriptApprovalNote$DescriptorImpl
```

Also there is a new Jetty error log which is likely a bug after the Jetty update in JFR's master branch

```
2020-04-13T14:42:02.403Z] test_cwp_with_timeout
[2020-04-13T14:42:02.403Z] 2020-04-13 14:42:02.098+0000 [id=1]	WARNING	o.e.jetty.webapp.WebAppContext#doStart: Failed startup of context o.e.j.w.WebAppContext@1033576a{Jenkins v2.222.1,/jenkins,file:///app/jenkins/,UNAVAILABLE}{/app/jenkins}
[2020-04-13T14:42:02.403Z] java.lang.IllegalStateException: No config path set
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.security.PropertyUserStore.loadUsers(PropertyUserStore.java:237)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.security.PropertyUserStore.doStart(PropertyUserStore.java:308)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.util.component.ContainerLifeCycle.addBean(ContainerLifeCycle.java:384)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.util.component.ContainerLifeCycle.addBean(ContainerLifeCycle.java:313)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.util.component.ContainerLifeCycle.updateBean(ContainerLifeCycle.java:864)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.security.HashLoginService.setUserStore(HashLoginService.java:126)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.security.HashLoginService.doStart(HashLoginService.java:176)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:117)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.security.SecurityHandler.doStart(SecurityHandler.java:405)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.security.ConstraintSecurityHandler.doStart(ConstraintSecurityHandler.java:419)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:110)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.server.handler.ScopedHandler.doStart(ScopedHandler.java:120)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.server.session.SessionHandler.doStart(SessionHandler.java:504)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:110)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.server.handler.ScopedHandler.doStart(ScopedHandler.java:120)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.server.handler.ContextHandler.startContext(ContextHandler.java:883)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.servlet.ServletContextHandler.startContext(ServletContextHandler.java:356)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.webapp.WebAppContext.startWebapp(WebAppContext.java:1445)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.webapp.WebAppContext.startContext(WebAppContext.java:1409)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.server.handler.ContextHandler.doStart(ContextHandler.java:825)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.servlet.ServletContextHandler.doStart(ServletContextHandler.java:275)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.webapp.WebAppContext.doStart(WebAppContext.java:524)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.server.Server.start(Server.java:407)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:110)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.server.Server.doStart(Server.java:371)
[2020-04-13T14:42:02.403Z] 	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
[2020-04-13T14:42:02.403Z] 	at io.jenkins.jenkinsfile.runner.JenkinsLauncher.createWebServer(JenkinsLauncher.java:56)
[2020-04-13T14:42:02.403Z] 	at io.jenkins.jenkinsfile.runner.JenkinsEmbedder.newHudson(JenkinsEmbedder.java:317)
[2020-04-13T14:42:02.403Z] 	at io.jenkins.jenkinsfile.runner.JenkinsEmbedder.before(JenkinsEmbedder.java:188)
[2020-04-13T14:42:02.403Z] 	at io.jenkins.jenkinsfile.runner.JenkinsLauncher.before(JenkinsLauncher.java:105)
[2020-04-13T14:42:02.403Z] 	at io.jenkins.jenkinsfile.runner.JenkinsLauncher.launch(JenkinsLauncher.java:69)
[2020-04-13T14:42:02.403Z] 	at io.jenkins.jenkinsfile.runner.App.run(App.java:13)
[2020-04-13T14:42:02.403Z] 	at io.jenkins.jenkinsfile.runner.bootstrap.Bootstrap.run(Bootstrap.java:307)
[2020-04-13T14:42:02.403Z] 	at io.jenkins.jenkinsfile.runner.bootstrap.Bootstrap.main(Bootstrap.java:127)
```

**Review by jglick — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
No new issues reported by CI. I will go ahead and merge it. One test is still broken, work in progress

**Pull request merged**